Makefile

```diff
@@ -19,9 +19,10 @@ pyang/xpath_parsetab.py: pyang/xpath_parser.py
 	python -m pyang.xpath_parser
 
 test: lint
-	(cd test; $(MAKE) test)
+	which xsltproc || (echo "Please run pip install -r dev-requirements.txt";exit 1)
```

Comment:
I don't think this is a good idea; there are other tools that need to be installed, and we can assume that developers do this.  If anything, we can add documentation about what a developer needs to do.

Reply:
This point is that if xsltproc isn't there, then the developer didn't read the documentation.